packages/components/src/components/popover/popover.tsx

```diff
@@ -52,18 +56,30 @@ export class Popover {
   private popover: HTMLDivElement;
   private button: HTMLButtonElement;
 
+  private isNative: boolean = false;
+  private parentTableElement: HTMLElement;
+
   public connectedCallback(): void {
     addDocumentEventListener(this);
+    this.detectNativeCase();
   }
 
   public componentShouldUpdate(newVal: unknown, oldVal: unknown): boolean {
     return hasPropValueChanged(newVal, oldVal);
   }
 
   public componentDidRender(): void {
-    if (this.open) {
-      // calculate / update position only possible after render
-      updatePopoverStyles(this.host, this.spacer, this.popover, this.direction, this.theme);
+    if (this.isNative && this.spacer?.matches(':popover-open')) {
+      addNativeScrollAndResizeListeners(this.host, this.parentTableElement, this.spacer);
```

Comment:
Do we need to remove these at some point? Or what happens if they are applied, then component is removed from DOM and then I scroll or resize?

Reply:
They get removed without an error, so do we still need some cleanup in disconnectedCallback?